### PR TITLE
docs: fix images sizes on logo wall pattern

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Corrects image sizes on logo wall pattern documentation

## Testing Instructions

1.

### CODE REVIEW

File: docs/patterns/logo-wall/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ A logo wall is a visual arrangement of logos representing various brands, compan
 <div class="grid">
   <div>
     <h3>Bordered</h3>
-    <uxdot-example>
+    <uxdot-example width-adjustment="1010px">
```
Comment (adamjohnson) [inline docs/patterns/logo-wall/index.md:28]: For this image, the intrinsic size is listed as `1120x110`. Needs updated here and on the `` tag.

File: docs/patterns/logo-wall/index.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ A logo wall is a visual arrangement of logos representing various brands, compan
   </div>
   <div>
     <h3>Borderless</h3>
-    <uxdot-example>
+    <uxdot-example width-adjustment="1010px"s>
```
Comment (adamjohnson) [inline docs/patterns/logo-wall/index.md:37]: For this image, the intrinsic size is listed as `1113x110`. Needs updated for `<uxdot-example>` and the `` tag.

File: docs/patterns/logo-wall/style.md [hunk 3]
```diff
@@ -61,23 +61,23 @@ Logo walls define layout rules for spacing logos, and their styles.
  - Logos are evenly spaced.
  - Logos are in a container with 16px of padding on all sides and a corner radius of 3px.
 
-<uxdot-example width-adjustment="560px">
+<uxdot-example width-adjustment="1113px">
   <img src="../style-space-group-bordered.png"
       alt="Example of a bordered logo wall with spacing values."
-      width="560"
-      height="55">
+      width="1113"
```
Comment (adamjohnson) [inline docs/patterns/logo-wall/style.md:67]: Intrinsic size: `1120x110`.

File: docs/patterns/logo-wall/guidelines.md [hunk 4]
```diff
@@ -29,19 +29,19 @@ logos.
...
 
 ### Borderless
 
-<uxdot-example>
+<uxdot-example width-adjustment="1120px">
   <img src="../guidelines-variants-borderless.png"
       alt="An example of a borderless logo wall with 6 logos."
-      width="556"
+      width="1120"
       height="55">
```
Comment (adamjohnson) [inline docs/patterns/logo-wall/guidelines.md:45]: Intrinsic size: `1113x110`—don't forget to adjust `` height too.

**Review by adamjohnson — CHANGES_REQUESTED** [review]
Just some image size changes/updates.

File: docs/patterns/logo-wall/index.md (hunk 1 above)
Comment (zeroedin) [reply docs/patterns/logo-wall/index.md:28]: `1010px` is the maximum size that `uxdot-example` has room for given desktop layout with padding.   I updated the `width` attribute on the `` tag.

My assumption here is that setting the `width-adjustment` on the uxdot-example will decrease possibility of any CLS, but the `max-width: 100%` might already be handling that on its on given the intrinsic size is larger then the available space.  🤔

**Review by adamjohnson — APPROVED** [review]
Logos get tighter margins

**Pull request merged**